cosmos/scheduler/cmd/scheduler.go

```diff
@@ -45,7 +52,23 @@ func (s *Scheduler) Run() {
 	}
 }
 
-func (s *Scheduler) configureIngestionSecret() error {
+func (s *Scheduler) healthCheck() {
+	health := func(w http.ResponseWriter, req *http.Request){
+		ctx := context.Background()
+		err := s.MongodbClient.Ping(ctx, readpref.Nearest())
+		if err != nil{
+			log.Println("MongoDB healthcheck failed:", err)
+			http.Error(w, "Internal error, could not contact MongoDB", 500)
+		}
```

Comment:
If err is nil, an OK response should be written, the request might hang otherwise

Reply:
When I tested this with curl, I got a `HTTP/1.1 200 OK` response which is why I left it out. However if you think it's better to be explicit then I can add it?

Reply:
Explicit 100% guaranteed better :)